Sources/System/Errno.swift

```diff
@@ -20,13 +20,12 @@ public struct Errno: RawRepresentable, Error, Hashable, Codable {
   @_alwaysEmitIntoClient
   public init(rawValue: CInt) { self.rawValue = rawValue }
 
-  @_alwaysEmitIntoClient
   private init(_ raw: CInt) { self.init(rawValue: raw) }
```

Comment:
Is there a reason to keep the private init in this case?

Reply:
I fumbled a conflict resolution. Will fix.